src/uwtools/resources/jsonschema/ioda.jsonschema

```diff
@@ -0,0 +1,53 @@
+{
+  "properties": {
+    "ioda": {
+      "additionalProperties": false,
+      "properties": {
+        "configuration_file": {
+          "additionalProperties": false,
+          "anyOf": [
+            {
+              "required": [
+                "base_file"
+              ]
+            },
+            {
+              "required": [
+                "update_values"
+              ]
+            }
+          ],
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "minProperties": 1,
+              "type": "object"
+            }
+          },
+          "type": "object"
+        },
+        "execution": {
+          "$ref": "urn:uwtools:execution"
```

Comment:
In the document prepared for `ioda` it was mentioned the driver should need serial execution.
```suggestion
        "execution": {
          "$ref": "urn:uwtools:execution-serial"
```

Reply:
This is an excellent point. I ended up running it with `srun -n 1` commands. Perhaps this is the better way, though.